src/common/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -231,6 +232,7 @@ bool ov::pass::CommonOptimizations::run_on_model(const std::shared_ptr<ov::Model
     REGISTER_PASS(manager, ConvertAvgPool14ToAvgPool1)
     REGISTER_PASS(manager, ConvertEmbeddingBagOffsets15ToEmbeddingBagOffsetsSum3)
     REGISTER_PASS(manager, ConvertEmbeddingBagPacked15ToEmbeddingBagPackedSum3)
+    REGISTER_PASS(manager, ConvertScatterNDUpdate15ToScatterNDUpdate3)
```

Comment:
should we always downgrade if no reduction? I think we now convert even for CPU.
Shall we implement smth on GPU and other platforms supporting only version 3?

Reply:
For Reduction::None, both v3 and v15 use same implementation so it shouldn't matter. Similarly ScatterElelementsUpdate has downgrade transform enabled while both CPU and GPU are enabled. But if it is an issue I can disable it in CPU

Note: currently both v3 and v15 have unspecified order of applying updates for duplicated indices - adding this functionality caused performance regressions on CPU side with possible further implementation issues on GPU side, so it was postponed for now. So, v15 with reduction::None is identical to old v3 version 

As for supporting GPU reductions, if there is need for it please let me know, I'll create tickets and include it in planning

Reply:
ScatterNDUpdate-15 is not used by frontends or transformations yet, so those changes shouldn't have impact on currently supported models. As the negative indices required constant input to be detected, disabling downgrade for CPU seems reasonable.

Reply:
I'll disable downgrade for CPU, but negative indices shouldn't affect CPU since support for them was added in PR 17219 to v3.
However, in other plugins like GPU, it doesn't seem like they are supported, so it may cause some issues in runtime when negative indices will be present